### ISSUE TITLE
Region sky circle contains

### ISSUE THREAD

**Issue by cdeil** [gammapy/gammapy#1327]
I noticed that this test fails, because `CircleSkyRegion` in `regions` now needs a `wcs`:
```
__________________________________________________________ TestEventListHESS.test_region ___________________________________________________________

self = <gammapy.data.tests.test_event_list.TestEventListHESS object at 0x1170ec080>

    def test_region(self):
        pos = SkyCoord(81, 21, unit='deg', frame='icrs')
        radius = Angle(1, 'deg')
        circ = CircleSkyRegion(pos, radius)
    
>       idx = circ.contains(self.events.radec)
E       TypeError: contains() missing 1 required positional argument: 'wcs'

gammapy/data/tests/test_event_list.py:44: TypeError
```

@joleroi - if you're available, I'd like to talk to you briefly about this. I'm not sure anymore what we discussed here, and what the plan was for sky circle and polygon regions. Should we change Gammapy to always pass a `wcs` when working with regions? Or convert to pixel regions ourselves?

**Comment by joleroi** [comment]
@cdeil In this case it's not obvious to me, because there is no ``wcs`` object at hand. So I would go for converting the event coordinates manually

**Comment by cdeil** [comment]
There's two things:

1. this test is easy to update (or remove), it doesn't actually test an API that couples Gammapy to regions, it just passes a SkyCoord to a regions API. So this could be done quickly, maybe we should and close this issue.

2. there's the question of how the Gammapy API should link to regions, and if it's somewhere by sky coords / sky regions, or only to pixel regions. This requires a review of what we have and want, i.e. you and I should probably sit together a bit and discuss.

**Comment by joleroi** [comment]
Fixed by #1363
